runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -248,9 +254,14 @@ public static class CorfuRuntimeParameters extends RuntimeParameters {
         int cacheConcurrencyLevel = 0;
 
         /*
-         * Sets expireAfterAccess and expireAfterWrite in seconds.
+         * Sets expireAfterAccess and expireAfterWrite for the AddressSpaceView cache in seconds.
          */
         long cacheExpiryTime = Long.MAX_VALUE;
+
+        /*
+         * Sets expireAfterAccess and expireAfterWrite for the MVOCache in seconds.
+         */
+        long mvoCacheExpiryTime = 300;
```

Comment:
I don't think we have enough data to support this optimization and for the sake of simplicity I would suggest just letting the LRU doing the eviction.

Reply:
Removed.